docs/parse_argparse.py

```diff
@@ -250,44 +253,48 @@ def link_main(rst_scripts_dirpath):
             if "main" in fname:
                 continue
             fname = os.path.splitext(fname)[0]
-            lines += f"\n\t {fname}"
+            lines += f"\t{fname}\n"
         with open(main_rst_fpath, "a") as fh:
             fh.write(lines)
 
 
-def write_usage_file(rst_scripts_dirpath):
+def write_usage_toctree(rst_scripts_dirpath):
     """
     Creates usage.rst and writes the directories in its toctree
 
     Parameters
     ----------
     rst_scripts_dirpath : string
-        path to the directory where script.rst files will be written,
-        e.g. '</path/to/docs/source/scripts/directory>'
+        path to scripts directory in docs,
+        e.g. '</path/to/docs/source/scripts>'
     """
     usage_fpath = os.path.join(rst_scripts_dirpath, "usage.rst")
     frmt_header = "="*len("usage")
-    text = f"{frmt_header}\nUsage\n{frmt_header}\n\n.. toctree::\n"
-    text += f"\t :maxdepth: 3\n\t :caption: Table of Contents\n\n"
-    if not os.path.exists(usage_fpath):
+    header = f"{frmt_header}\nUsage\n{frmt_header}\n\n.. toctree::\n"
+    header += f"\t :maxdepth: 3\n\t :caption: Table of Contents\n\n"
+    if os.path.exists(usage_fpath):
+        return
+    dir_names = ""
+    for dirname in os.listdir(rst_scripts_dirpath):
+        dirpath = os.path.join(rst_scripts_dirpath, dirname)
```

Comment:
Is it necessary to construct the full path to the directory just to check whether it is a directory?

Reply:
Yes, as explained in https://github.com/KwanLab/Autometa/pull/45#discussion_r419424572